docs/content/docs/getting-started/install-porter.md

```diff
@@ -114,6 +115,29 @@ You will need to create a [PowerShell Profile][ps-link] if you do not have one.
 iwr "https://cdn.porter.sh/latest/install-windows.ps1" -UseBasicParsing | iex
 ```
 
+## Air-gapped Install
+
+To run the installation script in an air-gapped environment, create an air-gapped installation package with the `bundle-linux.sh` script. Note that mixins are not currently included in the air-gapped-bundle.
+
+```sh
+curl -L https://raw.githubusercontent.com/getporter/porter/main/scripts/bundle/bundle-linux.sh | bash
+```
+
+Then extract and install Porter.
+
+```sh
+tar -xzf porter-air-gapped-install-latest.tar.gz -C .
+cd porter-air-gapped-install-latest
+bash install-bundle.sh
+```
+
+Note: The `install-bundle.sh` script starts mongo. If mongo ever has to be restarted use the following command.
+
+```
+docker run --name porter-mongodb-docker-plugin -d -p 27018:27017 -v mongodb_data:/data/db --restart always mongo:4.0-xenial
+```
```

Comment:
I've added a note here about mongo and how to restart it if it is stopped for any reason.

Reply:
Thanks looks very good